dotnet/src/Connectors/Connectors.OpenAI/Connectors.OpenAI.csproj

```diff
@@ -29,6 +29,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.AI.OpenAI" />
+    <PackageReference Include="Azure.AI.OpenAI"  VersionOverride="1.0.0-beta.17" />
```

Comment:
Why are we doing this?

Reply:
Temporary to allow the branch to build.  I expect this to dissappear once the AI connector work moves forward and I am able to merge to that.

Reply:
Update - I believe this project has recently (just) been removed in the AI connector migration.